app/components/general/components/bc-task-list/macro.njk

```diff
@@ -0,0 +1,30 @@
+{% macro bcTaskList(params) %}
+  <ol data-test-id="task-list" class="bc-c-task-list">
+    {% for task in params.taskList %}
+      {% set taskLoop = loop %}
```

Comment:
what's the purpose of this? why can't we just use `task`

Reply:
to remember the outer loop to be able to use loop.index in the inner loop

Reply:
👍